examples/value.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Noise-rs Developers.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An example of using value noise
+
+extern crate noise;
+
+use noise::{value2, value3, value4, Seed, Point2};
+
+mod debug;
+
+fn main() {
+    debug::render_png("value2.png", &Seed::new(0), 1024, 1024, scaled_value2);
+    debug::render_png("value3.png", &Seed::new(0), 1024, 1024, scaled_value3);
+    debug::render_png("value4.png", &Seed::new(0), 1024, 1024, scaled_value4);
+    println!("\nGenerated value2.png, value3.png and value4.png");
```

Comment:
@Razaekel: note that this is reintroducing the log message to an example - you'll have to fix that in #104 once if we merge this first